action.yml

```
@@ -75,6 +85,17 @@ runs:
          if ${{ inputs.load-all }} = true then
             LoadAllPackages();
          fi;
+
+         OutputLogTo();
+         CloseStream(output_stream);
+
+         # Treat warnings as errors if specified
+         if ${{ inputs.warnings-as-errors }} = 'true' ; then
```

Is that semicolon there legal? Does this all have to be GAP code or is it some weird hybrid?

---

Good spot. Once GitHub evaluates all of the `${{ ... }}` stuff, this should compile to normal GAP code, so the semicolon shouldn't be there.
```suggestion
         if ${{ inputs.warnings-as-errors }} = 'true' then
```